PetAdoption-iOS/PetAdoption-iOS/Controllers/PetListingDetailVC.swift

```diff
@@ -0,0 +1,271 @@
+//
+//  PetListingDetailVC.swift
+//  PetAdoption-iOS
+//
+//  Created by Keli'i Martin on 8/25/16.
+//  Copyright © 2016 Code For Orlando. All rights reserved.
+//
+
+import UIKit
+import PetAdoptionTransportKit
+
+enum Sections: Int
+{
+    case description = 0
+    case features = 1
+    case personality = 2
+    case adoptionInfo = 3
+}
+
+class PetListingDetailVC: UITableViewController
+{
+    ////////////////////////////////////////////////////////////
+    // MARK: - Typealiases
+    ////////////////////////////////////////////////////////////
+
+    typealias DictOfOptionalStrings = Dictionary<String, String?>
+    typealias ArrayofDicts = Array<DictOfOptionalStrings>
+
+    ////////////////////////////////////////////////////////////
+    // MARK: - IBOutlets
+    ////////////////////////////////////////////////////////////
+
+    @IBOutlet weak var imageContainerScrollView: UIScrollView!
+    @IBOutlet var imageContainerViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var pageControl: UIPageControl!
+
+    ////////////////////////////////////////////////////////////
+    // MARK: - Properties
+    ////////////////////////////////////////////////////////////
+
+    var descriptionSection = ArrayofDicts()
```

Comment:
Yeah, I was thinking about switching from using dictionaries to structs...  I'll update that